### ISSUE TITLE
Display UptimeKuma Status on any Service

### ISSUE THREAD

**Issue by mzalgaonker** [bastienwirtz/homer#689]
**Is your feature request related to a problem? Please describe.**
It would be awesome if we could display the UptimeKuma status of any service on the homer main page.

**Describe the solution you'd like**
I want to be able to display the UptimeKuma status on any service that UptimeKuma is monitoring.  For example, I am selfhosting several services: UptimeKuma and Jenkins.  

There is a status page setup in UptimeKuma that monitors my Jenkins service.

It would be cool to be able to display this UptimeKuma status on the Jenkins service on the Homer page.

**Describe alternatives you've considered**
I could not think of an alternative.  When I read that homer supported UptimeKuma, I thought this feature is included and did not realize it is limited to only the UptimeKuma service.

**Comment by giuliolibrando** [comment]
Agree, that would be amazing. Maybe just a simple dot near the name/icon (green,red)
